dev/lock.yml

```diff
@@ -85,28 +75,16 @@ dev:
       '(\s*"@typegraph\/sdk"\s*:\s*"npm:@typegraph\/sdk@).+(",?)': METATYPE_VERSION
       '(\s*"@typegraph\/sdk\/"\s*:\s*"npm:/@typegraph\/sdk@).+(\/",?)': METATYPE_VERSION
   lock:
-    GHJK_VERSION: 2725af8
-    GHJK_ACTION_VERSION: 2e8bbf084060a18828338a7cdd43fde6feb2a3cc
-    PYTHON_VERSION: 3.8.18
-    POETRY_VERSION: 1.7.0
-    PROTOC_VERSION: v24.1
+    GHJK_VERSION: 6a9ecda
+    GHJK_ACTION_VERSION: 318209a9d215f70716a4ac89dbeb9653a2deb8bc
```

Comment:
I wonder if this yml should not be used directly in deno and we integrate the lock.ts as ghjk module directly?

Reply:
Yeah, that's a good idea. It could go into standard library of sorts for port and ghjk.ts authors. I'll tackle it right away.